frontend/src/stimulus/controllers/dynamic/work-packages/date-picker/preview.controller.ts

```diff
@@ -364,7 +364,6 @@ export default class PreviewController extends DialogPreviewController {
     }
   }
 
-  // called from inputs defined in the date_picker/date_form.rb
```

Comment:
Why did you remove that comment?

Reply:
This file doesn't exist anymore. I will change it to date_picker/date_form_component.rb 👍